demo/admin/src/App.tsx

```diff
@@ -57,69 +58,71 @@ class App extends React.Component {
-                                                    >
-                                                        <React.Fragment>
-                                                            <GlobalStyle />
-                                                            <ContentScopeProvider>
-                                                                {({ match }) => (
-                                                                    <Switch>
-                                                                        {/* @TODO: add preview to contentScope once site is capable of contentScope */}
-                                                                        <Route
-                                                                            path={`${match.path}/preview`}
-                                                                            render={(props) => <SitePreview {...props} />}
-                                                                        />
-                                                                        <Route
-                                                                            render={() => (
-                                                                                <MasterLayout
-                                                                                    headerComponent={MasterHeader}
-                                                                                    menuComponent={MasterMenu}
-                                                                                >
-                                                                                    <MasterMenuRoutes menu={masterMenuData} />
-                                                                                </MasterLayout>
-                                                                            )}
-                                                                        />
-                                                                    </Switch>
-                                                                )}
-                                                            </ContentScopeProvider>
-                                                            <ErrorDialogHandler />
-                                                        </React.Fragment>
-                                                    </CmsBlockContextProvider>
-                                                </SnackbarProvider>
-                                            </DndProvider>
-                                        </RouterBrowserRouter>
-                                    </MuiThemeProvider>
-                                </LocaleProvider>
-                            </IntlProvider>
-                        </DamConfigProvider>
-                    </SitesConfigProvider>
-                </CurrentUserProvider>
-            </ApolloProvider>
+            <MlServiceContext.Provider value={{ enabled: true }}>
```

Comment:
Please sue the MlServiceProvider you added to the library for this

Reply:
I don't see the value of this provider.

Reply:
i wish, we only have providers that have a purpose :)

Reply:
This is currently missing in Demo, but needed (discussed internally). I was thinking about hard coding it, similar to ContentTranslationProvider. But we could think about having a general feature flag solution

Reply:
can you give an example how this hard coding could look like?

Reply:
> can you give an example how this hard coding could look like?

```tsx
function MyContentGenerationProvider() {
  const scope = useContentScope()

  ...

  return (
    <ContentGenerationServiceProvider enabled={scope.domain === "main"}>
      ...
    </ContentGenerationServiceProvider>
  )
}
```

Something like this?